### ISSUE TITLE
[Bitbucket] LFS settings

### ISSUE THREAD

**Issue by tuxmaster5000** [atlassian-api/atlassian-python-api#364]
Please add an function to set/get the git LFS status of an repo.

**Comment by gonchik** [comment]
Hi! 
Could you provide, please, a use case when do you need automate that one so often?

Cheers,
Gonchik Tsymzhitov

**Comment by tuxmaster5000** [comment]
Yes, when generating repos via script, LFS are disabled by default, so every repo must edit by hand.
The complete use case are repos for rpm package management like it is used by centos. Only teh control files are tracked via git native. The big binary files, are stored in the lfs area. So the checkout is very fast and the repo are very small, because in every branch only the "pointer" to the binary files must be tracked.

**Comment by gonchik** [comment]
https://jira.atlassian.com/browse/BSERV-8935

**Comment by gabyx** [comment]
Please add this.

**Comment by gonchik** [comment]
`    def get_lfs_repo_status(self, project_key, repo):
        url = f"rest/git-lfs/admin/projects/{project_key}/repos/{repo}/enabled"
        return self.get(url)

    def set_lfs_repo_status(self, project_key, repo, enable=True):
        url = f"rest/git-lfs/admin/projects/{project_key}/repos/{repo}/enabled"
        if enable:
            return self.put(url)
        else:
            return self.delete(url)`
